build/Common.ruleset

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="New Relic .NET Agent Rules" Description="New Relic .NET Agent ruleset for src and tests directories." ToolsVersion="16.0">
```

Comment:
We might want to update these names/descriptions to be specific to the telemetry SDK repo.

Reply:
Pushed up some new names.